src/ActionList/Group.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import {ItemProps} from './Item'
+
+export interface GroupProps extends React.ComponentPropsWithoutRef<'div'> {
+  renderItem?: (props: ItemProps) => JSX.Element
+}
+
+export function Group(props: GroupProps): JSX.Element {
+  return <div data-component="ActionList.Group" {...props} />
```

Comment:
Not sure how this works. I'm surprised TypeScript permits spreading an object containing `renderItem` onto a `<div>` element.

Reply:
Good catch, thanks! I’ll avoid applying unsupported attributes to HTML elements 👍

Reply:
Fixed in https://github.com/primer/components/pull/1078/commits/78ed7353bcefda20998b5a53e802e9504d3e76bb!